setup.py

```diff
@@ -46,6 +47,6 @@
     },
     test_requires=['pytest'],
     package_data={
-        'openapi': ['src/ratbag_emu/openapi/ratbag-emu.yaml'],
+        'ratbag_emu': ['openapi/ratbag-emu.yaml'],
```

Comment:
this seems unrelated to the "move src/main.py to __main__.py"

It's best to have one change per commit